docs/content/providers/kubernetes-ingress.md

```diff
@@ -529,6 +529,29 @@ providers:
 --providers.kubernetesingress.nativeLBByDefault=true
 ```
 
+### `strictPrefixMatching`
```

Comment:
Please could you add this documentation to our new reference documentation page here: https://github.com/traefik/traefik/blob/master/docs/content/reference/install-configuration/providers/kubernetes/kubernetes-ingress.md?plain=1#L39